target/linux/ramips/image/mt7621.mk

```diff
@@ -326,6 +326,20 @@ define Device/elecom_wrc-1167ghbk2-s
 endef
 TARGET_DEVICES += elecom_wrc-1167ghbk2-s
 
+define Device/elecom_wrc-1750gs
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 11264k
+  DEVICE_VENDOR := ELECOM
+  DEVICE_MODEL := WRC-1750GS/GSV
+  IMAGES += factory-gs.bin factory-gsv.bin
+  IMAGE/factory-gs.bin := $$(sysupgrade_bin) | check-size | \
+	elecom-gst-factory WRC-1750GS 0.00
+  IMAGE/factory-gsv.bin := $$(sysupgrade_bin) | check-size | \
+	elecom-gst-factory WRC-1750GSV 0.00
```

Comment:
Hmm, the Pull Request is obviously correct in what it wants to do.
I just wonder whether we should start doing this "having separate factory but using same device afterwards" thing.

Essentially, you could do the same with wrc-1900gst and wrc-2533gst as well, and then just have one board for all of them and four factory images. This is essentially what we did with ar71xx and then the router itself never knew which device it actually was.

So, I tend to say if the vendor really uses different strings to identify the devices, we should do the same and make this two devices. This would be in line with how we currently handle e.g. TP-Link devices in ath79, where we also have otherwise similar devices that just differ by one hardware id.

I'd still keep this in one commit, just add two DTS files and two device sections in mt7621.mk

Despite, one could also consider to create a mt7621_elecom_wrc-gst-16m.dtsi, which would contain the partitions so we don't have to repeat them 3/4 times.

Reply:
> ... you could do the same with wrc-1900gst and wrc-2533gst as well, and then just have one board for all of them and four factory images ...
I tend to say if the vendor really uses different strings to identify the devices, we should do the same and make this two devices ...

Indeed, I agree with that. OK, I'll split GS/GSV.